src/pages/workspaces/migration/BillingProjectParent.ts

```diff
@@ -1,21 +1,88 @@
 import _ from 'lodash/fp';
-import { ReactNode } from 'react';
-import { div, h } from 'react-hyperscript-helpers';
+import pluralize from 'pluralize';
+import { ReactNode, useState } from 'react';
+import { div, h, span } from 'react-hyperscript-helpers';
 import Collapse from 'src/components/Collapse';
+import { ButtonOutline } from 'src/components/common';
+import { Ajax } from 'src/libs/ajax';
 import colors from 'src/libs/colors';
-import { BillingProjectMigrationInfo } from 'src/pages/workspaces/migration/migration-utils';
+import { reportErrorAndRethrow } from 'src/libs/error';
+import * as Utils from 'src/libs/utils';
+import {
+  BillingProjectMigrationInfo,
+  errorIcon,
+  getBillingProjectMigrationStats,
+  inProgressIcon,
+  successIcon,
+} from 'src/pages/workspaces/migration/migration-utils';
 import { WorkspaceItem } from 'src/pages/workspaces/migration/WorkspaceItem';
+import { useMemo } from 'use-memo-one';
 
 interface BillingProjectParentProps {
   billingProjectMigrationInfo: BillingProjectMigrationInfo;
 }
 
 export const BillingProjectParent = (props: BillingProjectParentProps): ReactNode => {
+  const [migrateStarted, setMigrateStarted] = useState<boolean>(false);
+
+  const migrationStats = useMemo(() => {
+    return getBillingProjectMigrationStats(props.billingProjectMigrationInfo);
+  }, [props.billingProjectMigrationInfo]);
+
+  const renderErrorSummary = () => {
+    return migrationStats.errored === 0
+      ? ''
+      : span({ style: { paddingLeft: '0.5rem', paddingRight: '0.5rem' } }, [
+          errorIcon,
+          span({ style: { paddingLeft: '0.5rem', color: colors.danger() } }, [
+            `${pluralize('Migration', migrationStats.errored, true)} Failed`,
+          ]),
+        ]);
+  };
+
+  const renderMigrationSummary = () => {
+    return Utils.cond(
+      [
+        migrationStats.workspaceCount === migrationStats.succeeded,
+        () =>
+          span({}, [
+            successIcon,
+            span({ style: { paddingLeft: '0.5rem', paddingRight: '0.5rem' } }, [
+              migrationStats.workspaceCount === 1
+                ? '1 Workspace Migrated'
+                : `All ${migrationStats.workspaceCount} Workspaces Migrated`,
+            ]),
+          ]),
+      ],
+      [
+        migrationStats.inProgress > 0,
+        () =>
+          span({}, [
+            inProgressIcon,
+            span({ style: { paddingLeft: '0.5rem', paddingRight: '0.5rem' } }, [
+              `${pluralize('Workspace', migrationStats.inProgress, true)} Migrating`,
+              migrationStats.errored > 0 && ', ',
+              renderErrorSummary(),
```

Comment:
When we have a workspace migration in progress, I decided to NOT show the number of already completed migrations in order to reduce clutter. My thought is that the most important bits of information are if workspaces are still migrating and if any of them errored.